cannot find symbol  startForeground(FOREGROUND_SERVICE_TYPE_MICROPHONE, notification);

# Bug report

- [x] I've checked the [example](https://github.com/react-native-webrtc/react-native-callkeep/tree/master/example) to reproduce the issue.

At the time building the app, the package throwing error saying 

```
/Users/ad/MY COOL APP/node_modules/react-native-callkeep/android/src/main/java/io/wazo/callkeep/VoiceConnectionService.java:59: error: cannot find symbol
import static android.content.pm.ServiceInfo.FOREGROUND_SERVICE_TYPE_MICROPHONE;
^
  symbol:   static FOREGROUND_SERVICE_TYPE_MICROPHONE
  location: class ServiceInfo

/Users/ad/MY COOL APP/node_modules/react-native-callkeep/android/src/main/java/io/wazo/callkeep/VoiceConnectionService.java:251: error: cannot find symbol
        startForeground(FOREGROUND_SERVICE_TYPE_MICROPHONE, notification);
                        ^
  symbol:   variable FOREGROUND_SERVICE_TYPE_MICROPHONE
  location: class VoiceConnectionService
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
2 errors
```

- Reproduced on:
 - [x] Android
 - [ ] iOS

## Description


## Steps to Reproduce
Install the package, follow the installation process https://github.com/react-native-webrtc/react-native-callkeep/blob/master/docs/android-installation.md

## Versions
    - Callkeep: "^4.0.0"
    - React Native: "0.63.2"

--- a/README.md
+++ b/README.md
@@ -830,6 +830,7 @@ Since iOS 13, you'll have to report the incoming calls that wakes up your applic
 ## Android 11
 
 Since Android 11, your application [requires to start a foregroundService](https://developer.android.com/about/versions/11/privacy/foreground-services) in order to access the microphone in background.
+You'll need to upgrade your `compileSdkVersion` to `30` to be able to use this feature.
 
 You have to set the `foregroundService` key in the [`setup()`](#setup) method and add a `foregroundServiceType` in the [`AndroidManifest` file](docs/android-installation.md#android-common-step-installation).
 

--- a/android/src/main/java/io/wazo/callkeep/Constants.java
+++ b/android/src/main/java/io/wazo/callkeep/Constants.java
@@ -18,4 +18,6 @@ public class Constants {
     public static final String EXTRA_CALLER_NAME = "EXTRA_CALLER_NAME";
     // Can't use telecom.EXTRA_DISABLE_ADD_CALL ...
     public static final String EXTRA_DISABLE_ADD_CALL = "android.telecom.extra.DISABLE_ADD_CALL";
+
+    public static final int FOREGROUND_SERVICE_TYPE_MICROPHONE = 128;
 }

--- a/android/src/main/java/io/wazo/callkeep/VoiceConnectionService.java
+++ b/android/src/main/java/io/wazo/callkeep/VoiceConnectionService.java
@@ -56,7 +56,6 @@ import java.util.Set;
 import java.util.UUID;
 import java.util.stream.Collectors;
 
-import static android.content.pm.ServiceInfo.FOREGROUND_SERVICE_TYPE_MICROPHONE;
 import static io.wazo.callkeep.Constants.ACTION_AUDIO_SESSION;
 import static io.wazo.callkeep.Constants.ACTION_ONGOING_CALL;
 import static io.wazo.callkeep.Constants.ACTION_CHECK_REACHABILITY;
@@ -65,6 +64,7 @@ import static io.wazo.callkeep.Constants.EXTRA_CALLER_NAME;
 import static io.wazo.callkeep.Constants.EXTRA_CALL_NUMBER;
 import static io.wazo.callkeep.Constants.EXTRA_CALL_UUID;
 import static io.wazo.callkeep.Constants.EXTRA_DISABLE_ADD_CALL;
+import static io.wazo.callkeep.Constants.FOREGROUND_SERVICE_TYPE_MICROPHONE;
 
 // @see https://github.com/kbagchiGWC/voice-quickstart-android/blob/9a2aff7fbe0d0a5ae9457b48e9ad408740dfb968/exampleConnectionService/src/main/java/com/twilio/voice/examples/connectionservice/VoiceConnectionService.java
 @TargetApi(Build.VERSION_CODES.M)

